README.md

```diff
@@ -107,7 +107,7 @@ await withTaskGroup(of: Void.self) { taskGroup in
 Once a client is running, queries can be sent to the server. This is straightforward:
 
 ```swift
-let rows = try await client.query("SELECT id, username, birthday FROM users")
+let rows = try await client.query("SELECT id, username, birthday FROM users", logger: Logger(label: "my-example-logger"))
```

Comment:
I don't think this is true:
```
    @discardableResult
    public func query(
        _ query: PostgresQuery,
        logger: Logger? = nil,
        file: String = #fileID,
        line: Int = #line
    ) async throws -> PostgresRowSequence {
```
@Joannis can you validate that?

Reply:
@Joannis PING!

Reply:
I must've pulled in an old version maybe? You're right

Reply:
I think you talk about PostgresConnection, which implements PostgresDatabase.